oslabs-beta/GraphPulse
# Dt/splashpage

Issue Type

[ ] Bug
[ x ] Feature
[ ] Tech Debt

Description
Updated splash page to include Github/Linkedin links and a Demo GIF.

Ticket/Card Name
n/a

Steps to Validate Feature

At splash page check the demo
Click the linkedin/github icons to redirect to each contributor's links.
